what4/what4.cabal

```diff
@@ -204,16 +205,36 @@ test-suite bvdomain_tests
   type: exitcode-stdio-1.0
   default-language: Haskell2010
 
-  hs-source-dirs: test
+  hs-source-dirs: test, test/QC
   main-is: BVDomTests.hs
 
-  other-modules:
+  other-modules:  VerifyBindings
 
   build-depends: base
                , parameterized-utils
                , tasty >= 0.10
                , tasty-quickcheck >= 0.10
                , QuickCheck
```

Comment:
We should do a little bit of investigating to find some reasonable package bounds for QC.  As I recall QC version 2 made a lot of breaking changes. I don't remember when the `Gen` monad was introduced, for example.

Reply:
Good point.  We can probably re-introduce the original bounds and then see if they could be loosened or if they are what we want.

Reply:
Turns out `chooseInt` and `chooseInteger` were both added in 2.14; they are simply higher-performance variants of the generic `choose` method.  We instead just use `choose` loosen the bounds, I think.

Reply:
I was just looking at that as well.  The older `choose` is less performant, but I don't think we are inordinately slow here, so broadening the bounds is probably a better benefit.